keyboards/keyboardio/atreus/keymaps/rwijtvliet/keymap.c

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 ,2020  Keyboard.io ,Inc
+//
+// Ruud Wijtvliet
+// TODO:
+// . toggle to layer on three-key combo
+// . prevent rolling errors
+//
+// good tutorial:
+// https://precondition.github.io/home-row-mods
+
+#include "users/rwijtvliet/keymap_partial.c"
+#include "atreus.c"
```

Comment:
c files should not be included.  They should be added to the SRC, in the rules.mk.